### PR TITLE
Update verification process for helm charts and policies

### PR DESCRIPTION
**Pull request by viccuad**

### CODE REVIEW

File: docs/security/verifying-kubewarden.md [hunk 1]
```diff
@@ -109,3 +117,19 @@ Verified OK
 
 You can inspect the cert signature yourself to see that indeed was authenticated
 via GitHub OIDC, and performed in our GitHub Actions workflows.
+
+## Policies
+
+Policies maintained by the Kubewarden team are also signed via Sigstore. As seen for
```
Comment (divya-mohan0209) [inline docs/security/verifying-kubewarden.md:123]: Slightly rephrasing it to make the sentence sound idiomatic.
```suggestion
Policies maintained by the Kubewarden team are also signed using the Sigstore project. Similar to
```

**Review by divya-mohan0209 — COMMENTED** [review]
Just a small nit. Otherwise Lgtm!

File: docs/security/verifying-kubewarden.md [hunk 2]
```diff
@@ -109,3 +117,19 @@ Verified OK
 
 You can inspect the cert signature yourself to see that indeed was authenticated
 via GitHub OIDC, and performed in our GitHub Actions workflows.
+
+## Policies
+
+Policies maintained by the Kubewarden team are also signed using the Sigstore project. Similar to
+usual container images, one can verify them using `cosign`:
+```
+vic@viccuad2 1 ~/suse/kw/kubewarden-controller (main $%>)$ COSIGN_EXPERIMENTAL=1 cosign verify ghcr.io/kubewarden/policies/verify-image-signatures:v0.2.0
```
Comment (viccuad) [inline docs/security/verifying-kubewarden.md:126]:
```suggestion
COSIGN_EXPERIMENTAL=1 cosign verify ghcr.io/kubewarden/policies/verify-image-signatures:v0.2.0
```

**Review by divya-mohan0209 — APPROVED** [review]
Lgtm, thanks!

**Review by jvanz — APPROVED** [review]

**Pull request merged**